pkg/errors/types.go

```diff
@@ -28,18 +28,24 @@ var (
 	// ErrConflict indicates that entity already exists.
 	ErrConflict = New("entity already exists")
 
-	// ErrCreateEntity indicates error in creating entity or entities
+	// ErrCreateEntity indicates error in creating entity or entities.
 	ErrCreateEntity = New("failed to create entity in the db")
 
-	// ErrViewEntity indicates error in viewing entity or entities
+	// ErrViewEntity indicates error in viewing entity or entities.
 	ErrViewEntity = New("view entity failed")
 
-	// ErrUpdateEntity indicates error in updating entity or entities
+	// ErrUpdateEntity indicates error in updating entity or entities.
 	ErrUpdateEntity = New("update entity failed")
 
-	// ErrRemoveEntity indicates error in removing entity
+	// ErrRemoveEntity indicates error in removing entity.
 	ErrRemoveEntity = New("failed to remove entity")
 
-	// ErrScanMetadata indicates problem with metadata in db
+	// ErrScanMetadata indicates problem with metadata in db.
 	ErrScanMetadata = New("failed to scan metadata in db")
+
+	// ErrJSONErrKey indicates response body did not contain erorr message.
+	ErrJSONErrKey = New("response body expected error message json key not found")
+
+	// ErrUnknown indicates that an unknown error was found in the response body.
+	ErrUnknown = New("unknown error")
```

Comment:
These are internal errors and should not be exported. You can use separate `var` block for them.